packages/eslint-plugin/tests/rules/no-empty-interface.test.ts

```diff
@@ -148,5 +148,27 @@ type Foo<T> = Bar<T>
         },
       ],
     },
+    {
+      filename: 'test.d.ts',
+      code: `
+declare module FooBar {
+  type Baz = typeof baz;
+  export interface Bar extends Baz {}
+}
+      `,
+      errors: [
+        {
+          messageId: 'noEmptyWithSuper',
+          line: 4,
+        },
+      ],
```

Comment:
make sure you test the suggestion fixer.
EG:
https://github.com/typescript-eslint/typescript-eslint/blob/d3d70a39abfe208dfe5092d2e05069957b6cedf2/packages/eslint-plugin/tests/rules/prefer-nullish-coalescing.test.ts#L282-L292